recipes/gliner/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "gliner" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/gliner-{{ version }}.tar.gz
+  sha256: 2c00570066a016d3ec682fc0d1aa0c61e7318873343662233c6994ac37fb2a10
+
+build:
+  noarch: python  # won't work on windows due to missing pytorch win build
```

No need for this comment. The recipe and the artifact will work if an end user can provide pytorch for Windows vai another channel. That is the beauty of noarch packages.

```suggestion
  noarch: python
```